src/screens/SearchAndGo/components/HeaderTitle.js

```diff
@@ -0,0 +1,39 @@
+import { Image, StyleSheet, Text, View } from "react-native";
+import React from "react";
+
+const HeaderTitle = ({title}) => {
+  return (
+    <View style={styles.headerView}>
+      <View style={styles.imageContainer} />
+      <Text style={styles.textStyle}>{title}</Text>
+      <View style={styles.imageContainer}>
+        <Image
```

Comment:
This arrow back button should take you back to previous screen.
use <Pressable> instead of View
also use on onPress function 
() => navigation.goBack()

Reply:
I made the changes and fix all the conflicts, please review